medicines/pars-upload/src/pages/_app.js

```diff
@@ -8,9 +8,25 @@ import { signIn, getAccount } from '../auth/authPopup'
 
 function App({ Component, pageProps }) {
   const [auth, setAuth] = useState(null)
+  const [authError, setAuthError] = useState(null)
 
   const triggerSignIn = () => {
-    signIn().then(setAuth)
+    signIn()
+      .then(setAuth)
+      .catch((e) => handleAuthError(e.toString()))
+  }
+
+  function handleAuthError(error) {
+    if (error.startsWith('ClientAuthError:')) {
+      error = error.slice(16)
+
+      if (error.includes('Login_In_Progress')) {
+        error =
+          'Login popup is already open in another window, perhaps it is behind this window.'
+      }
+
+      setAuthError(error)
+    }
```

Comment:
I think if there's an unknown error we want to show an error message still. Perhaps at the bottom we could add something roughly along the lines of 

```suggestion
    } else {
      setAuthError("Unknown error")
    }
```

Reply:
fair